ajax/libs/jquery-easy-ticker/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-easy-ticker",
+  "filename": "jquery.easy-ticker.min.js",
+  "version": "2.0.0",
+  "description": "jQuery Easy ticker plugin",
+  "homepage": "http://www.aakashweb.com/jquery-plugins/easy-ticker/",
```

Comment:
I think `https://www.aakashweb.com/jquery-plugins/easy-ticker/` will be better, thanks!